db/data/sample/clouds.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
This file and the other `data/sample/*.yml` files appear to have been blindly copied into this repository.